controllers/cleanup_controller.go

```diff
@@ -66,6 +69,11 @@ func (r *CleanupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 
 	logger.Info("storagesystem instance found")
 
+	// TODO: this check should be removed in 4.20
```

Comment:
Please add the context and need for this check on the comment

Reply:
done.